packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -1202,6 +1324,41 @@ export default function makeKernelKeeper(kernelStorage, kernelSlog) {
     return makeUpgradeID(nextID);
   }
 
+  function addTerminatedVat(vatID) {
```

Comment:
Suggested clarifying rename:
```suggestion
  function setVatTerminated(vatID) {
```

Reply:
Hm, yeah, but I think `markVatAsTerminated` or `markVatTerminated` might be better.

Something like "setVatTerminated" reads (to me) as setting a boolean flag named "vatTerminated", which might be appropriate on a `VatKeeper` (which is already scoped to a single vatID), but which would really want a kernelKeeper backend, which would need a name, and I didn't feel like adding two functions for this one feature.